assets/overlays/openstack-manila/patches/node_add_driver.yaml

```diff
@@ -57,6 +57,12 @@ spec:
             - name: MANILA_SHARE_PROTO
               value: NFS
           volumeMounts:
+            - name: config-manilaplugin
+              mountPath: /etc/kubernetes/config
+              readOnly: true
+            - name: secret-manilaplugin
+              mountPath: /etc/kubernetes/secret
```

Comment:
Purely a nice to have, but `clouds.yaml` are typically found in `/etc/openstack`. I know you're aligning with Cinder here - which is good - but I wonder if we could use that here?

Note: I am planning on changing this in Cinder too